### PR TITLE
Citing pyDARN

### PR DESCRIPTION
**Pull request by mts299**
As pyDARN is two PR's away (this one and #33 ) from a release branch being created then released, I thought I would add a section on how to cite pyDARN as well SuperDARN. 

Please review your name in the citation BibTeX and thumbs up it if you are okay with your name's initials.

This PR will not stay open long after someone approves it so please give it a glance.

### CODE REVIEW

File: docs/user/citing.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+If pyDARN contributes to a project that leads to a scientific publication, please acknowledge this fact by citing the software and SuperDARN.  
+
+# How to cite SuperDARN
+
+Super Dual Auroral Radar Network (SuperDARN) is a network of 36 radars 20 institutions, to cite general SuperDARN you can use: 
+
+- Greenwald, R.A., Baker, K.B., Dudeney, J.R. et al. Space Sci Rev (1995) 71: 761. [doi:10.1007/BF00751350](https://doi.org/10.1007/BF00751350)
+
+For the general achievements of the SuperDARN Network, you can read these papers:
```
Comment (aburrell) [inline docs/user/citing.md]: Recommend "When referencing the general or regional achievements of SuperDARN, we recommend these review papers:"

File: docs/user/citing.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+```
+
+## DOI's 
+
+v1.0.0
+Currently in development  
+
+## Acknowledgements
+
+pyDARN uses other python libraries and the SuperDARN communities help.
```
Comment (aburrell) [inline docs/user/citing.md]: "communities" should be "community's"

**Comment by mts299** [comment]
I am looking to merge this PR today so I can use it in #33. 

Please check it out ASAP and approve or thumbs up it if you are good with how the citing page looks. 
https://pydarn.readthedocs.io/en/docs-citing/user/citing/

**Comment by aburrell** [comment]
It looks ok apart from the first comment I made where you have an awkward sentence that reads "Network Network" through the SuperDARN acronym.  The important bits all look good.

**Comment by mts299** [comment]
fixed @aburrell, thank you for pointing that out again.

**Comment by mts299** [comment]
Going to merge this in now, if you have comments on changes there will be a chance in the #33 PR as I want to link to the citing section there as well.

**Pull request merged**